packages/puppeteer-core/src/api/Target.ts

```diff
@@ -18,6 +18,7 @@ export enum TargetType {
   BACKGROUND_PAGE = 'background_page',
   SERVICE_WORKER = 'service_worker',
   SHARED_WORKER = 'shared_worker',
+  WORKER = 'worker',
```

Comment:
I think we need to either start supporting this type for CDP targets (which could be a breaking change) or declare the target type as Other for BiDi workers for now.